src/presentation/pages/History.vue

```diff
@@ -0,0 +1,139 @@
+<template>
+  <ThreeColsLayout data-dimensions="large">
+    <div :class="$style['history']">
+      <div :class="$style['history__page-header']">
+        <Heading
+          :level="1"
+          :class="$style['title']"
+        >
+          {{ noteTitle }}
+        </Heading>
+        <Heading
+          :level="2"
+          :class="$style['subtle']"
+        >
+          {{ t('history.title') }}
+        </Heading>
+      </div>
+
+      <div :class="$style['history__container']">
+        <Container
+          :class="$style['history-items']"
+        >
+          <Row
+            v-for="(historyRecord, index) in noteHistory"
+            :key="historyRecord.id"
+            :title="historyRecord.user.name"
+            :subtitle="parseDate(new Date(historyRecord.createdAt))"
+            :has-delimiter="noteHistory !== null && index !== noteHistory?.length - 1"
+            :class="$style['history-items__row']"
+          >
+            <template #left>
+              <Avatar
+                :src="historyRecord.user.photo"
+                :username="historyRecord.user.name"
+              />
+            </template>
+            <template #right>
+              <Button
+                secondary
+                icon="Search"
```

Comment:
there is no icon in figma

Reply:
![image](https://github.com/user-attachments/assets/5bdcbe81-6193-43ff-8619-7ce61bc3ca00)
this is temporary solution for Eye icon (as i can see it is not implemented in current version of codex icons)

Reply:
removed